CHANGELOG.md

```diff
@@ -6,8 +6,28 @@ All notable changes to the [Camunda Modeler](https://github.com/camunda/camunda-
 
 ___Note:__ Yet to be released changes appear here._
 
+## 4.4.0
+
+### General
+
+* `FEAT`: enable `Backspace` on MacOS for element removal ([#1989](https://github.com/camunda/camunda-modeler/issues/1989))
+* `FEAT`: enable `SHIFT + click` for multi-selection ([#1964](https://github.com/camunda/camunda-modeler/issues/1964))
+* `FIX`: correct switching between diagram and XML ([#1925](https://github.com/camunda/camunda-modeler/issues/1925))
+* `CHORE`: capture deployment and process variables in telemetry events
```

Comment:
Is there a reason we don't reference https://github.com/camunda/camunda-modeler/issues/1971 here?

Reply:
Just did not want to mention more internals for already internal things.

Reply:
:+1: